samples/hello++/main.cpp

```diff
@@ -0,0 +1,35 @@
+#include <xboxrt/debug.h>
+#include <pbkit/pbkit.h>
+#include <hal/xbox.h>
+
+class NumberClass{
+private:
+  int hiddenValue = 0;
+public:
+  NumberClass() {hiddenValue = 5;};
+  NumberClass(int v) : hiddenValue(v) {};
+  int getValue() { return hiddenValue; };
+};
+
+int main() {
+  int ret = pb_init();
+  if (ret != 0) {
+    XSleep(2000);
+    return -1;
+  }
+
+  pb_show_debug_screen();
+
+  NumberClass defaultFive;
+  NumberClass customSix(6);
+  
+  while (true) {
```

Comment:
This requires `#include <stdbool.h>` (or `#include <cstdbool>` eventually). We should fix it when doing a cleanup pass on all samples.

It currently depends on one of the other headers including it.